plugins/flashrom/fu-flashrom-device.c

```diff
@@ -57,6 +86,10 @@ fu_flashrom_device_set_quirk_kv (FuDevice *device,
 		fu_device_set_metadata_integer (device, "PciBcrAddr", tmp);
 		return TRUE;
 	}
+	if (g_strcmp0 (key, "Programmer") == 0) {
```

Comment:
FlashromProgrammer please, we're prefixing the quirk names now.

Reply:
Done. Thank you!